docs/coordinates/satellites.rst

```diff
@@ -0,0 +1,86 @@
+.. include:: references.txt
+
+.. _astropy-coordinates-satellites:
+
+Working with Earth satellites using Astropy Coordinates
+*******************************************************
+
```

Comment:
If you are feeling up for it, perhaps you could write a few sentences here as an introduction / overview of earth satellite coordinate frame conventions, with some references?

Reply:
done...